third_party/CMakeLists.txt

```diff
@@ -19,6 +16,15 @@ if(${EFFCEE_ENABLE_TESTS})
 endif()
 
 if (IS_DIRECTORY ${EFFCEE_RE2_DIR})
+
+  if ("${MSVC}")
```

Comment:
are saying we should put re2 under `dxc/external` rather than `dxc/external/effcee/third_party/re2` ?